src/server/routes/api/document/index.js

```diff
@@ -1220,6 +1221,59 @@ const checkApiKey = (apiKey) => {
   }
 };
 
+// TODO RPN
+const getNovelInteractions = async doc => { // eslint-disable-line
+  return [];
+};
+
+const emailRelatedPaperAuthors = async doc => {
+  const getContact = paper => _.get(paper, ['pubmed', 'authors', 'contacts', 0]);
+
+  const hasContact = paper => getContact(paper) != null;
+
+  const sendEmail = async (paper, novelIntns) => {
+    const contact = getContact(paper);
+    const email = contact.email[0];
+    const name = contact.name;
+
+    // TODO RPN enable to prevent duplicate sending
+    // doc.relatedPapersNotified(true);
+
+    const mailOpts =  await msgFactory(EMAIL_TYPE_REL_PPR_NOTIFICATION, doc, {
+      to: 'maxkfranz@gmail.com', // send to one of us for testing
```

Comment:
```suggestion
      to: 'support@biofactoid.org', // send to one of us for testing
```

@jvwong, would you prefer to use the info or support address for testing this?

Reply:
Maybe I should make this a config, I wouldn't want all this noise going to the support.

Reply:
like `EMAIL_ADMIN_ADDR`

Reply:
The idea is to test this on unstable.  It would use the user's email address on prod.  It needs to go to a shared address if more than one person is able to test it.